samples/authorization/csm_authorization_proxy_server_v230.yaml

```diff
@@ -85,6 +85,16 @@ spec:
           sentinel: sentinel
           redisReplicas: 5
 
+        - name: jwt
+          # JWT signing secret configuration:
+          # If using a CSI driver, specify the SecretProviderClass.
+          # Otherwise, a default Kubernetes secret will be used for JWT signing secret.
+          jwtSigningSecretProviderClass:
+             # The name of the Kubernetes secret created by the CSI driver
+            - jwtSigningSecretName: ""
+              # The key in the secret that holds the Redis username
+              jwtSigningSecretKey: secret
```

Comment:
Can this be set to an empty string and we can assign a default value to it?

Reply:
This refers to the key name inside the secret store, not the actual secret string. We can rename it though to something more descriptive to avoid confusion.